sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/append_blob_client.hpp

```diff
@@ -117,6 +117,17 @@ namespace Azure { namespace Storage { namespace Blobs {
     Azure::Core::Response<Models::CreateAppendBlobResult> Create(
         const CreateAppendBlobOptions& options = CreateAppendBlobOptions()) const;
 
+    /**
+     * @brief Creates a new 0-length append blob. The content keeps unchanged if the blob already
+     * exists.
+     *
+     * @param options Optional parameters to execute this function.
+     * @return A CreateAppendBlobResult describing the newly created append blob. Null if the blob
```

Comment:
I think Null is a bit misleading to our customers. How about `Raw response if the blob already exists.`

Also, we agreed to remark that not checking if the response has value before accessing it may cause process to crash. Please also document that.

Reply:
it always returns the raw response

Reply:
plus it aligns with .Net https://github.com/Azure/azure-sdk-for-net/blob/dfbc7aa9ee5f053d2d84f85d4d142152886f53ed/sdk/storage/Azure.Storage.Blobs/src/BlobContainerClient.cs#L822